setup.sh

```diff
@@ -4,14 +4,26 @@ RPI_SETUP_DIR="$( pwd )"
 
 # Disable the built-in audio output so there is only one audio
 # device in the system
-sudo sed -i -e 's/dtparam=audio=on/#dtparam=audio=on/' /boot/config.txt
+sudo sed -i -e 's/^dtparam=audio=on/#dtparam=audio=on/' /boot/config.txt
 
 # Enable the i2s device tree
 sudo sed -i -e 's/#dtparam=i2s=on/dtparam=i2s=on/' /boot/config.txt
 
+# Enable the I2C device tree
+sudo raspi-config nonint do_i2c 1
+sudo raspi-config nonint do_i2c 0
+
+# Set the I2C baudrate to 100k
+sudo sed -i -e '/^dtparam=i2c_arm_baudrate/d' /boot/config.txt
+sudo sed -i -e 's/dtparam=i2c_arm=on$/dtparam=i2c_arm=on\ndtparam=i2c_arm_baudrate=100000/' /boot/config.txt
+
 echo "Installing Raspberry Pi kernel headers"
 sudo apt-get install -y raspberrypi-kernel-headers
 
+echo "Installing the Python3 packages and related libs"
+sudo apt-get install python3-matplotlib
```

Comment:
Suggest using `-y` to avoid interrupting top level `auto_install.sh` (xmos/vocalfusion_3510_avs_setup). Alternatively, document the requirement to type yes in the top level readme